src/Popover/Popover.styles.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+import { Box } from '../Box';
+import { Button } from '../Button';
+
+export const StyledPopoverContent = styled(Box)`
+  background-color: ${({ theme }) => theme.palette.white.main};
+  border: 1px solid ${({ theme }) => theme.palette.neutral[300]};
+  border-radius: 4px;
+  padding: 16px;
+`;
+
+export const StyledCloseButton = styled(Button)`
+  padding: 0px;
+  height: 16px;
+  width: 16px;
```

Comment:
Could you replace all `px` unit to rem? You can use the utility:
```
import { pxToRem } from '../shared/mixins';
```
```suggestion
  width: ${pxToRem(16)}rem;
```